src/main/scala/tech/cryptonomic/conseil/metadata/UnitTransformation.scala

```diff
@@ -69,4 +94,14 @@ class UnitTransformation(overrides: MetadataConfiguration) {
         currencySymbolCode = overrideAttribute.flatMap(_.currencySymbolCode)
       )
     }
+
+  private def logDifferences(paths: List[Path], overriddenPaths: List[Path]): Unit = {
+    paths
+      .filterNot(overriddenPaths.toSet)
+      .foreach(logger.warn("""There're missing metadata overrides for "{}"""", _))
```

Comment:
Personal taste:
I'd rather make a single string list with `.mkString` and then issue a single `log.warn` with that, both here and immediately after

Reply:
Is having paths that are not overridden considered something to get worried about?
If that's not the case I'd suggest using `logger.info` and write the message in softer tones (e.g. "these paths were not overridden: ...")

Reply:
1. I would prefer to have a single log too but it would require some fancy collecting so I decided to do it that way in order to keep code simple. 
2. According to the task description, it's something we need to worry about. @anonymoussprocket requested explicitly that it should be `warn` and `error`.